WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -756,9 +760,13 @@ private extension ProductsViewController {
             message: .init(string: message),
             image: .emptyProductsTabImage,
             details: details,
-            buttonTitle: buttonTitle) { [weak self] button in
-            self?.addProduct(sourceView: button)
-        }
+            buttonTitle: buttonTitle,
+            onTap: { [weak self] button in
+                self?.addProduct(sourceView: button)
+            },
+            onPullToRefresh: { [weak self] refreshControl in
+                self?.pullToRefresh(sender: refreshControl)
+            })
```

Comment:
nit: similar to the orders PR, can we also add this to the empty results view from filters in `createNoProductsMatchFilterConfig`?

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/7030/commits/a0061ed3a8a1722d2bef087094fc4a8e487c8331